src/chapters/setup-intro-data-types/data-types.rst

```diff
@@ -1,25 +1,22 @@
 Data Types
 ===========
 
-Java handles values and variables differently than JavaScript, but in a similar
-manner to `TypeScript chapter <https://education.launchcode.org/intro-to-professional-web-dev/chapters/typescript/variables.html>`__.
-
 Static vs. Dynamic Typing
 -------------------------
 
 .. index:: ! dynamically typed, ! statically typed
 
-JavaScript is a **dynamically typed** language, which means that a variable or
-parameter can refer to a value of any data type (string, number, object, etc.)
-at any time. When the variable is used, the interpreter figures out what type
-it is and behaves accordingly.
-
 Java is a **statically typed** language. When a variable or parameter is
 declared in a statically typed language, the data type for the value must be
 specified. Once the declaration is made, the variable or parameter cannot refer
 to a value of any other type.
 
-For example, this is legal in JavaScript:
+Statically typed languages differ from **dynamically typed** languages. In dynamically
```

Comment:
Since we want the focus to be on Java, I think the flow works better if it moves from dynamic typing to static.  I'd keep the original paragraph order but modify the phrasing in the first.  Something like:

In a **dynamically typed** programming language (like JavaScript or Python), a variable or parameter can refer to a value of any data type (string, number, object, etc.) at any time. When the variable is used, the interpreter figures out what type it is and behaves accordingly.